.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Do you want to commit this file?

Reply:
I don't see anything wrong with it but since these are autogenerated it's fine. I deleted it.